extensions/vault/runtime/src/main/java/io/quarkus/vault/pki/ConfigURLsOptions.java

```diff
@@ -0,0 +1,25 @@
+package io.quarkus.vault.pki;
+
+import java.util.List;
+
+/**
+ * Options for configuring URLs
```

Comment:
be consistent with adding a dot at the end of these comments.

Reply:
👍